jerry-core/api/jerry.c

```diff
@@ -379,7 +379,7 @@ jerry_parse (const jerry_char_t *resource_name_p, /**< resource name (usually a
              size_t source_size, /**< script source size */
              uint32_t parse_opts) /**< jerry_parse_opts_t option bits */
 {
-#if defined JERRY_DEBUGGER && !defined JERRY_DISABLE_JS_PARSER
+#if defined JERRY_DEBUGGER && ENABLED (JERRY_PARSER)
```

Comment:
Are you planning to change the `JERRY_DEBUGGER` macro as well?

Reply:
The JERRY_DEBUGGER will have its own PR as it "crosses" directories (jerry-core, jerry-ext, etc.)